packages/common/src/items/index.ts

```diff
@@ -16,3 +16,4 @@ export * from "./get-structured-license";
 export * from "./slugs";
 export * from "./normalize-solution-template-item";
 export * from "./setItemThumbnail";
+export * from "./registerBrowserApp";
```

Comment:
does this _need_ to be part of the external API? Is anyone ever going to use this fn on it's own, or just `registerSiteAsApplication()`?

Reply:
needed b/c it's re-exported in sites package